source/Halibut.Tests/TraceLogFileLogger.cs

```diff
@@ -85,30 +87,14 @@ public bool CopyLogFileToArtifacts()
             var rootDirectory = new DirectoryInfo(currentDirectory).Parent.Parent.Parent.Parent.Parent;
 
             var traceLogsDirectory = rootDirectory.CreateSubdirectory("artifacts").CreateSubdirectory("trace-logs");
-            var fileName = $"{testHash}.tracelog";
-
-            try
-            {
-                File.Copy(tempFilePath, Path.Combine(traceLogsDirectory.ToString(), fileName), true);
-                return true;
-            }
-            catch
-            {
-                return false;
-            }
+            return traceLogsDirectory;
         }
 
         public void Dispose()
         {
-            try
-            {
-                File.Delete(tempFilePath);
-            }
-            catch
-            {
-                // Best effort clean-up, but we don't want to
-                // fail the test because we couldn't delete this file
-            }
+            cancellationTokenSource.Cancel();
```

Comment:
No need to delete the log file.

but we do await the writing of logs, since we really want to know if writing logs doesn't work.